src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/BookingFacade.kt

```diff
@@ -54,7 +56,7 @@ class BookingFacade(
 
   fun create(bookingRequest: CreateVideoBookingRequest, createdBy: User): Long {
     require(createdBy is PrisonUser || availabilityService.isAvailable(bookingRequest)) {
-      if (bookingRequest.bookingType == BookingType.COURT) {
+      if (bookingRequest.bookingType == RequestBookingType.COURT) {
         "Unable to create court booking, booking overlaps with an existing appointment."
```

Comment:
Wondering why this is the only place which uses an alias of BookingType?

Reply:
Because this is was already using a BookingType object but taken from the model package.  We have one in Domain.Perhaps ModelBookingType would be better?